src/components/TodoContainer.js

```diff
@@ -1,98 +1,54 @@
-import axios from 'axios'
 import React, { useState, useEffect } from 'react'
 
 import TodosList from './TodoList'
 import Header from './Header'
 import InputTodo from './InputTodo'
 
+import { getTodosFromApi, deleteFromApi, addTodoToApi } from '../services/api'
+
 export const TodoContainer = () => {
-  const [state, setState] = useState({
-    data: {
-      todos: [],
-    },
-  })
+  const [todos, setTodos] = useState([{ id: 0, title: '', completed: false }])
+
+  /**
+   * Get 10 todo items from API when component is mounted
+   */
```

Comment:
I like to put the effects first on my component, makes it easier to see what's going on either on mount or on update